### ISSUE TITLE
vue cli4.04 快速原型开发 失败

### ISSUE THREAD

**Issue by hearsay316** [vuejs/vue-cli#4731]
### Version
4.0.4

### Environment info
```
Environment Info:

  System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz
  Binaries:
    Node: 10.16.3 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.7.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.17763.771.0
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
定义一个App.vue 内容为

```
<template>
    <div id="app">
        tgtgtg
    </div>
</template> 
```
执行vue serve 

### What is expected?
运行服务 

### What is actually happening?
(node:13948) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'version' of undefined
    at module.exports (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\cli-service-global\node_modules\@vue\cli-plugin-es
lint\index.js:18:27)
    at plugins.forEach (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\cli-service-global\node_modules\@vue\cli-service\
lib\Service.js:83:7)
    at Array.forEach (<anonymous>)
    at Service.init (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\cli-service-global\node_modules\@vue\cli-service\lib
\Service.js:81:18)
    at Service.run (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\cli-service-global\node_modules\@vue\cli-service\lib\
Service.js:220:10)
    at Object.exports.serve (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\cli-service-global\index.js:85:24)
    at Command.program.command.description.option.option.option.action (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\c
li\bin\vue.js:122:53)
    at Command.listener (C:\Users\111\AppData\Roaming\npm\node_modules\@vue\cli\node_modules\commander\index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)
(node:13948) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside
of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:13948) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections th
at are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by OBKoro1** [comment]
同样的问题

**Comment by sodatea** [comment]
Fixed in 4.0.5
